### DISCUSSION TITLE
how to make the facets under default channel to be available across all other channels

### DISCUSSION THREAD

**Discussion by ramkishore-yerubandi** [vendurehq/vendure#1226] [Q&A] [answered]
My use case needs to control the creation of facets under each individual channel. We would like create the facets under default channel and make them available for each channel so that they select the facets that comes from Default Channel. Is there a way we can achieve this?

**Comment by michaelbromley** [comment, marked as answer]
Currently we are lacking an `assignFacetsToChannel` mutation. The work-around would be to implement this mutation yourself, following the general guidelines for plugins found here: https://www.vendure.io/docs/plugins/plugin-examples/extending-graphql-api/

**Reply by ramkishore-yerubandi** [reply]
Thanks!
